source/scene.js

```diff
@@ -394,4 +394,11 @@ this.playVideo = function( src ) {
     playVideo( id );
 }
 
+this.enableBlocklyTab = function ( nodeID ) {
+    var node = this.findByID( this, nodeID );
```

Comment:
This is a costly find. I'd suggest either passing the node or the node name instead.